src/api/verify.ts

```diff
@@ -78,14 +83,86 @@ class VerifyAPI {
       expiresAt,
       expiresIn,
       passed,
+      failureReasons,
+      _id,
     } as RadarTrackVerifiedResponse;
 
     if (options.debug) {
       trackRes.response = response;
     }
 
+    lastToken = trackRes;
```

Comment:
token is the entire response?

Reply:
Yes. Deserialized to `RadarVerifiedLocationToken` in mobile SDKs